src/lib.rs

```diff
@@ -31,6 +31,7 @@ pub mod blockstream;
 pub mod blockstream_service;
 pub mod blocktree_processor;
 pub mod cluster_info;
+pub mod cluster_tests;
```

Comment:
Prefix your additions with`#[cfg(test)]`?

Reply:
@garious tests doesn't seem to build that way
```
   Compiling solana v0.12.0 (/home/aeyakovenko/solana)
error[E0432]: unresolved import `solana::cluster_tests`
 --> tests/local_cluster.rs:3:5
  |
3 | use solana::cluster_tests;
  |     ^^^^^^^^^^^^^^^^^^^^^ no `cluster_tests` in the root

error[E0432]: unresolved import `solana::local_cluster`
 --> tests/local_cluster.rs:4:13
  |
4 | use solana::local_cluster::LocalCluster;
  |             ^^^^^^^^^^^^^ could not find `local_cluster` in `solana`

error: aborting due to 2 previous errors
```

Reply:
yeah, `local_cluster` too of course

Reply:
@garious you mean this, right?
```
$ git diff
diff --git a/src/lib.rs b/src/lib.rs
index 7fad53fc..55154a8e 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -31,6 +31,7 @@ pub mod blockstream;
 pub mod blockstream_service;
 pub mod blocktree_processor;
 pub mod cluster_info;
+#[cfg(test)]
 pub mod cluster_tests;
 pub mod db_window;
 pub mod entry;
@@ -43,6 +44,7 @@ pub mod gossip_service;
 pub mod leader_confirmation_service;
 pub mod leader_schedule;
 pub mod leader_schedule_utils;
+#[cfg(test)]
 pub mod local_cluster;
 pub mod local_vote_signer_service;
 pub mod packet;
```

tests depends on lib built without #[cfg(test)] for some reason
```
   Compiling solana v0.12.0 (/home/aeyakovenko/solana)
error[E0432]: unresolved import `solana::cluster_tests`
 --> tests/local_cluster.rs:3:5
  |
3 | use solana::cluster_tests;
  |     ^^^^^^^^^^^^^^^^^^^^^ no `cluster_tests` in the root

error[E0432]: unresolved import `solana::local_cluster`
 --> tests/local_cluster.rs:4:13
  |
4 | use solana::local_cluster::LocalCluster;
  |             ^^^^^^^^^^^^^ could not find `local_cluster` in `solana`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.
error: Could not compile `solana`.
warning: build failed, waiting for other jobs to finish...
error: build failed
[Finished running. Exit status: 101]
```